src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double bucketCapacity; // in cubic meters
+    private double maxReach;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator`. Use private access modifiers to encapsulate the fields and provide getters and setters for accessing them.